app/controllers/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EventsController < ApplicationController
+  def index
+    @past_events = Event.find_past_events
+    @upcoming_events = Event.find_upcoming_events
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def show
+    @event = Event.find(params[:id])
+    @creator = User.find(@event.creator_id)
+    @event_invitees = Invitation.list_invitees(@event)
+    @invitees = Invitation.name_invitees(@event_invitees)
+  end
+
+  def create
+    @event = Event.new(event_params)
+    @user = User.find(current_user.id)
```

Comment:
This can be simplified by writing @event = current_user.events.build(event_params)

Reply:
Thank you @ntwaliHeriter for your review.